src/components/Calculator.js

```diff
@@ -1,59 +1,50 @@
-          <button onClick={this.eventClicked} type="button" className="input">9</button>
-          <button onClick={this.eventClicked} type="button" className="input orange-btn">x</button>
-          <button onClick={this.eventClicked} type="button" className="input">4</button>
-          <button onClick={this.eventClicked} type="button" className="input">5</button>
-          <button onClick={this.eventClicked} type="button" className="input">6</button>
-          <button onClick={this.eventClicked} type="button" className="input orange-btn">-</button>
-          <button onClick={this.eventClicked} type="button" className="input">1</button>
-          <button onClick={this.eventClicked} type="button" className="input">2</button>
-          <button onClick={this.eventClicked} type="button" className="input">3</button>
-          <button onClick={this.eventClicked} type="button" className="input orange-btn">+</button>
-        </div>
-        <div className="last-row">
-          <button onClick={this.eventClicked} type="button" className="input">0</button>
-          <button onClick={this.eventClicked} type="button" className="input">.</button>
-          <button onClick={this.eventClicked} type="button" className="input orange-btn">=</button>
-        </div>
+  return (
+    <div className="calculator">
+      <div className="answer">
+        <h3 className="show-answer">
+          { input.total }
+          { input.operation }
+          { input.next }
+        </h3>
       </div>
-    );
-  }
+      <div className="calc-panel">
+        <button onClick={eventClicked} type="button" className="input">AC</button>
+        <button onClick={eventClicked} type="button" className="input">+/-</button>
+        <button onClick={eventClicked} type="button" className="input">%</button>
+        <button onClick={eventClicked} type="button" className="input orange-btn">÷</button>
+        <button onClick={eventClicked} type="button" className="input">7</button>
+        <button onClick={eventClicked} type="button" className="input">8</button>
+        <button onClick={eventClicked} type="button" className="input">9</button>
+        <button onClick={eventClicked} type="button" className="input orange-btn">x</button>
+        <button onClick={eventClicked} type="button" className="input">4</button>
+        <button onClick={eventClicked} type="button" className="input">5</button>
+        <button onClick={eventClicked} type="button" className="input">6</button>
+        <button onClick={eventClicked} type="button" className="input orange-btn">-</button>
+        <button onClick={eventClicked} type="button" className="input">1</button>
+        <button onClick={eventClicked} type="button" className="input">2</button>
+        <button onClick={eventClicked} type="button" className="input">3</button>
+        <button onClick={eventClicked} type="button" className="input orange-btn">+</button>
```

Comment:
**Optional Changes**
- [ ] There is too much repetition here on the button. You could store the button names in an array and use `array.map` to dynamically add them to the DOM. Let us try to make these changes even if they're optional here. It will make us adhere to the `DRY` principle.

Reply:
Thanks for the suggestion. I have added the following changes to my code. I am sending it for another review now 😄